.circleci/config.yml

```diff
@@ -791,7 +796,8 @@ workflows:
               - data8 image build
               - data100 image build
               - data101 image build
-              - astro image build
+              - data102 image build
+	      - astro image build
```

Comment:
could you check and see if this is a tab, and if so, remove it and use spaces

Reply:
Done